plugins/inventory/gcp_compute.py

```diff
@@ -422,9 +429,12 @@ def _add_hosts(self, items, config_data, format_items=True, project_disks=None):
         if self.get_option("hostnames"):
             hostname_ordering = self.get_option("hostnames")
 
+        if self.get_option("base_domain"):
```

Comment:
please remove this if. Otherwise the `base_domain` variable will not be found if it is not a set option, and crash on line 437.